sympy/assumptions/ask.py

```diff
@@ -729,6 +729,44 @@ def is_true(self):
         """
         return Predicate('is_true')
 
+    # XXX: empty_matrix or just empty?
+    @predicate_memo
+    def empty_matrix(self):
+        """
+        A matrix having at least one dimension equal to zero.
+
+        Examples
+        ========
+
+        >>> from sympy import Q, ask, MatrixSymbol
```

Comment:
There can also be empty `Matrix` objects. They should probably be included somewhere.